public/js/monitor.js

```diff
@@ -436,38 +484,42 @@ window.addEventListener("pageshow", function() {
   if (window.location.search.includes("utm_") && window.history.replaceState) {
     window.history.replaceState({}, "", window.location.toString().replace(/[?&]utm_.*/g, ""));
   }
-
-  if (document.forms) {
-    restoreInputs();
-  }
+  toggleMobileFeatures();
+  document.forms ? (restoreInputs(), addFormListeners()) : null;
```

Comment:
I had to read this a couple times to figure out what it was doing... Not sure if this is any clearer than the previous vanilla `if()` style:

```js
if (document.forms) {
  restoreInputs();
  addFormListeners();
}
```